modules/todos.js

```diff
@@ -0,0 +1,12 @@
+export default [
+  {
+    index: 1,
+    description: 'Practice Webpack correclty',
+    completed: true,
+  },
+  {
+    index: 3,
+    description: 'Watching mock interviews',
+    completed: false,
+  },
+];
```

Comment:
- [ ] **Required:** At the moment your to do list do not display according to the index, kindly fix by sorting them by their indexes. Eg. `Array.sort((a, b) => a.index - b.index);`.